util/image_viewer/spotfinder_frame.py

```diff
@@ -639,7 +639,9 @@ def draw_resolution_rings(self, unit_cell=None, space_group=None):
         # FIXME Currently assuming that all panels are in same plane
         p_id = detector.get_panel_intersection(beam.get_s0())
         if p_id == -1:
-            p_id = 0  # XXX beam doesn't intersect with any panels - is there a better solution?
+            p_id = (
+                0
+            )  # XXX beam doesn't intersect with any panels - is there a better solution?
```

Comment:
Would this comment be better moved to the line above to avoid this ugly formatting?

Reply:
Probably. Black did this (I didn't touch the line on purpose)